runtime-complexity/runtime.js

```diff
@@ -109,9 +109,9 @@ const timings = [
 
 main = function(document) {
 
-  var elem = document.getElementById("myChart");
-  var ctx = elem.getContext("2d");
-  var myChart = new Chart(ctx, {
+  const elem = document.getElementById("myChart");
+  const ctx = elem.getContext("2d");
+  let myChart = new Chart(ctx, {
```

Comment:
This should be `const` instead of `let`.
```suggestion
  const myChart = new Chart(ctx, {
```